great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -8,6 +8,7 @@
 )
 from great_expectations.rule_based_profiler.domain_builder import (
     CategoricalColumnDomainBuilder,
+    DomainBuilder,
```

Comment:
If only importing for the purposes of type hinting, please use the following pattern:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from great_expectations...
```

Reply:
Agreed